readthedocs/templates/projects/project_version_detail.html

```diff
@@ -31,6 +31,14 @@ <h2> Editing {{ version.slug }} </h2>
 
 
     <form method="post" action=".">
+      <p class="empty">
+        {% url 'projects_redirects' project.slug as redirects_url %}
+        {% blocktrans with redirects_url=redirects_url current_slug=version.slug %}
+          <strong>Warning!</strong> Changing the slug will break existing URLs.
+          If you change it, you may want to <a href="{{ redirects_url }}">create an exact redirect</a>
+          from <span>/en/{{ current_slug }}/$rest</span> to <span>/en/new-slug</span>.
```

Comment:
Not really sure if we want to make this _more real_, using current lang, single version, subproject, etc

Reply:
I think it's good enough.